.github/workflows/test-query-engine-driver-adapters.yml

```
@@ -25,24 +25,24 @@ jobs:
       fail-fast: false
       matrix:
         adapter:
-          - name: '@prisma/adapter-planetscale'
+          - name: 'planetscale (napi)'
             setup_task: 'dev-planetscale-js'
-          - name: '@prisma/adapter-pg (napi)'
+          - name: 'pg (napi)'
             setup_task: 'dev-pg-js'
-          - name: '@prisma/adapter-neon (ws) (napi)'
+          - name: 'neon (ws) (napi)'
             setup_task: 'dev-neon-js'
-          - name: '@prisma/adapter-libsql (Turso) (napi)'
+          - name: 'libsql (napi)'
```

```suggestion
          - name: 'libsql Turso (napi)'
```
to make it similar to the other one